src/rtf-plugins/fixture-managers/yarpplugin/YarpPluginFixture.h

```diff
@@ -29,10 +29,11 @@ class YarpPluginFixture : public RTF::FixtureManager {
     virtual bool setup(int argc, char** argv);
     virtual bool check();
     virtual void tearDown();
-    bool scanPlugins(yarp::os::ConstString name);
+    bool scanPlugin(yarp::os::ConstString name, yarp::os::ConstString type="");
```

Comment:
Unrelated to the patch, but since you changed the name, perhaps this method should be private? Or protected? I don't think it is useful to have it public...

By the way, just out of curiosity, why did you change `scanPlugins` to `scanPlugin`?